### PR TITLE
fix: ignore record does not exist error

### PR DESCRIPTION
**Pull request by froid1911**

### CODE REVIEW

File: src/storage/prisma.ts [hunk 1]
```diff
@@ -495,21 +495,16 @@ export class PrismaStorage implements IDriveStorage {
...
-                id_driveId: {
+        try {
+            await this.db.document.deleteMany({
+                where: {
                     driveId: drive,
                     id: id
                 }
-            },
-            include: {
-                operations: {
```
Comment (acaldas) [inline src/storage/prisma.ts:506]: We also need to delete operations and attachments right? Or deleteMany does this?

Comment (acaldas) [reply src/storage/prisma.ts:506]: Can you add an unit test to make sure everything is correctly deleted?

File: src/storage/prisma.ts [hunk 2]
```diff
@@ -495,21 +495,16 @@ export class PrismaStorage implements IDriveStorage {
...
-                    include: {
-                        attachments: true
-                    }
-                }
-            }
-        });
+            });
+        } catch (e) {
+            // do nothing
```
Comment (acaldas) [inline src/storage/prisma.ts:506]: We should ignore just that specific error, not all exceptions. Prisma usually adds an error code

File: src/storage/prisma.ts (hunk 1 above)
Comment (froid1911) [reply src/storage/prisma.ts:506]: the db is configured to delete also operations, attachments and syncUnits when a doc gets deleted (onCascade=delete). will provide a unit test

**Review by acaldas — APPROVED** [review]

**Pull request merged**